train/custom_datasets.py

```diff
@@ -213,11 +135,11 @@ def __getitem__(self, i) -> Dict[str, torch.Tensor]:
         if i in self.cached_data_dict:
             return self.cached_data_dict[i]
 
-        ret = prepare_messages_for_model(self.raw_data[i], self.tokenizer)
+        ret = prepare_training_inputs(self.raw_data[i], self.tokenizer)
         ret = {
-            "input_ids": ret["input_ids"],
-            "labels": ret["labels"],
-            "attention_mask": ret["attention_mask"],
+            "input_ids": ret[1]["input_ids"],
```

Comment:
Why is this ret[1] ? Can we find a more explicit way. If we cannot, we should at least add some clarification